examples/with-firebase/pages/index.js

```diff
@@ -26,6 +34,12 @@ export default function Home() {
       <main>
         <h1 className="title">Next.js w/ Firebase Client-Side</h1>
         <p className="description">Fill in your credentials to get started</p>
+
+        <button onClick={() => createUser()}>Create 'nextjs_user'</button>
```

Comment:
What happens if I click this multiple times?

Reply:
It's probably ignored by firestore after the first click

Reply:
But it'll trigger an update with every click, no matter how fast afaik.

Reply:
It's basically lacking a loading state.

Reply:
If you have more than one click, it will be overwritten with the latest data.
In this example, we are always sending the same data and it just doesn't seem to change.